(#2) Adds Initial Examples (#3)
chocolatey/ccm-deployment-examples: deployments/installing-a-package-simple/README.md

# Installing a Package (Simple)

## Purpose

> This deployment is a basic example of deploying a package using a simple deployment step.

Reviewer: ```suggestion
This Deployment Plan is a basic example of deploying a package using a simple deployment step.
```

> This deployment is a basic example of deploying a package using a simple deployment step.

Reviewer: ```suggestion
This Deployment Plan is a basic example of deploying a package using a simple deployment step.
```

> This Deployment Plan is a basic example of deploying a package using a simple deployment step.

Reviewer 2: ```suggestion
This Deployment Plan is a basic example of deploying a package using a simple Deployment Step.
```

> https://github.com/chocolatey/ccm-deployment-examples/blob/b79340df0363be0fcda3192828cef96095320d3d/deployments/installing-a-package-simple/Deployment.json#L1-L22

Reviewer 2: Does this embed it into this file?

Author: Yes, though when it's in a permanant branch I intend to embed only the latest version of main, which will mean that we won't need to update it. Can't do that until _something_ is merged, though.

Reviewer 3: @JPRuskin this isn't rendering correctly, let's take it out.

## Usage

> The only deployment step contained upgrades (installing if not present) the `Firefox` package ([available on the Chocolatey Community Repository](https://community.chocolatey.org/packages/firefox)).

Reviewer 3: I feel like this reads weird, but it could just be me?

> The only Deployment Step in this Deployment Plan upgrades (installing if not present) the `Firefox` package ([available on the Chocolatey Community Repository](https://community.chocolatey.org/packages/firefox)).

Reviewer 2: ```suggestion
The only Deployment Step in this Deployment Plan upgrades, or installs if not present, the `Firefox` package ([available on the Chocolatey Community Repository](https://community.chocolatey.org/packages/firefox)).
```

> Assuming this package was available on your sources, you will only need to set the target groups for the deployment.

Reviewer 3: Tell them how to quickly get the package if they don't have it?

> Assuming this package was available on your sources, you will only need to set the target groups for the deployment. If you don't have it available, you can download it using `choco download firefox --source="'https://community.chocolatey.org/api/v2/'"` and then upload it to your internal repository.

Reviewer 2: ```suggestion
Assuming this package was available on your sources, you will only need to set the Target Groups for the Deployment Step. If you don't have it available, you can download it using `choco download firefox --source="'https://community.chocolatey.org/api/v2/'"` and then upload it to your internal repository.
```

### Setting the Target Group

You will need to set a target group for the deployment. 

> To do so:

Reviewer 2: ```suggestion
You will need to set a Target Group for the Deployment Step:
```

> - Mouse-over the deployment step and click the `Edit` button:

Reviewer 2: ```suggestion
- Mouse-over the Deployment Step and click the `Edit` button.
```

    ![Editing a Deployment Step](/images/EditDeploymentStep.jpg)

- Navigate to the `Select Target Groups` panel

- Highlight your preferred groups and click the `>` button to move them into `Selected Groups`
    ![Selecting Target Groups](/images/SelectingTargetGroups.jpg)

> - Click `Save`

Reviewer 2: ```suggestion
- Navigate to the `Select Target Groups` panel.

- Highlight your preferred groups and click the `>` button to move them into `Selected Groups`.
    ![Selecting Target Groups](/images/SelectingTargetGroups.jpg)

- Click `Save`.
```

> You are now ready to [move your deployment to Ready and run it](https://docs.chocolatey.org/en-us/central-management/usage/website/deployments#move-to-ready).

Reviewer 2: ```suggestion
You are now ready to [move your Deployment Plan to Ready and run it](https://docs.chocolatey.org/en-us/central-management/usage/website/deployments#move-to-ready).
```
